packages/debian/openarm-can-utils.install

```diff
@@ -12,6 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-setup/configure_socketcan.sh usr/libexec/openarm-can
-setup/set_zero.sh usr/libexec/openarm-can
-setup/change_baudrate.py usr/libexec/openarm-can
+usr/libexec/*
```

Comment:
Can we omit `*` here?

```suggestion
usr/libexec/
```

Reply:
Thanks! I have confirmed we can omit it.
I’ve also fixed the other parts.

https://github.com/enactic/openarm_can/compare/3e0df06cd067e1d044558d18a8f2d44bc48b0085..bcd7aefb2d31a7eed732fd267abfa97d7d72db1a